person_capture/gui_app.py

```
@@ -7371,6 +7478,30 @@ def _save_hdr_sdr_screencap(
             pass
+        conversion = str(getattr(self.cfg, "hdr_sdr_conversion", "windows_wic") or "windows_wic").strip().lower()
+        if conversion in {"windows_wic", "wic", "paint", "paint_wic"} and tmp_ext == ".png":
+            # Paint-like path: exact HDR crop -> WIC color-managed PNG. This is
+            # deliberately not a fallback to the preview renderer; if WIC cannot
+            # decode/convert the HDR still, report that as the save failure.
+            tmp_hdr = out_path + ".tmp_hdr.avif"
+            try:
+                if os.path.exists(tmp_hdr):
+                    os.remove(tmp_hdr)
+            except Exception:
+                pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)